OasisDEX/oasis-borrow
# added proxy creation disabled flag

# [Disable proxy creation flag (for The Merge)](https://app.shortcut.com/oazo-apps/story/5955/ui-disable-proxy-creation-for-the-ethereum-merge)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
When ProxyCreationDisabled flag is on and user doesn't have existing proxy ,creation of new proxy should be temporary disabled with message as in story

 - Check when opening borrow vault 
 
![image](https://user-images.githubusercontent.com/6871923/189366962-1ae1513a-97a3-4bd9-9b04-7c47a55f44a6.png)

- check for multiply vault opening
![image](https://user-images.githubusercontent.com/6871923/189377849-ae559ffd-7b32-4a4b-b3b0-5aa7269811f4.png)

- check for aave vault (if will be released b4 the Merge)
- check for guni vaults (not sure how to do it locally)
